applications/luci-app-banip/htdocs/luci-static/resources/view/banip/overview.js

```diff
@@ -232,7 +232,7 @@ return view.extend({
 						'click': ui.createHandlerFn(this, function () {
 							return handleAction('restart');
 						})
-					}, [_('Apply & Restart')])
+					}, [_('Restart')])
```

Comment:
So yoy changed the Apply & Restart back to just Restart? Or this is a merging error